content/messages/messages-fr.yaml

```diff
@@ -0,0 +1,120 @@
+---
+switch_to: "Passez en"
+hi_there: "Bienvenue chez nous !"
```

Comment:
"Bienvenue" simplement non ? bienvenue chez nous me fait penser à l'émission de TF1...

Reply:
Didn't know about this TV show, I'll change. :)